modules/transactions/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -1285,8 +1284,6 @@ void testTwoPhaseResetOnEmptyNodes() throws Exception {
 
     @Test
     @ZoneParams(nodes = 7, replicas = 3, partitions = 1, consistencyMode = ConsistencyMode.HIGH_AVAILABILITY)
-    // TODO https://issues.apache.org/jira/browse/IGNITE-24144
-    @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "false")
```

Comment:
I would be grateful if you check also `@Disabled` tests regarding chain logic. They must fail with the same reason as they failed when colocation flag set to false. I've checked several, they pass, but I could miss something.

Reply:
Checked. For me tests failed with same exception both for colocation enabled and disabled. 
I'm talking about `testSecondResetRewritesUnfinishedFirstPhaseReset` and `testAssignmentsChainUpdatedOnAutomaticReset`